src/BotPlutusInterface/Balance.hs

```diff
@@ -116,14 +118,23 @@ balanceTxIO' ::
 balanceTxIO' balanceCfg pabConf ownPkh unbalancedTx =
```

Comment:
Due to change in how we calculate _minAda_ for `TxOuts`, this balancing algorithm doesn't handle the case where new `TxOut`(s) are created during balancing of Tx, should contain the _minAda_.

Reply:
Fixed!